openedx/features/edly/tests/test_utils.py

```diff
@@ -62,6 +76,13 @@ def _create_edly_sub_organization(self):
         """
         return EdlySubOrganizationFactory(lms_site=self.request.site)
 
+    def _get_course_creator_status(self, user):
+        """
+        Helper method to get user's course creator status
```

Comment:
nit: `.` at end of sentence https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings

Reply:
done